msd/SpeakerAward.py

```diff
@@ -0,0 +1,35 @@
+import json
+import azure.cognitiveservices.speech as speechsdk
+from dotenv import load_dotenv
+import os
+from pydub import AudioSegment
+
+class SpeakerAward():
+    
+    def __init__(self,diarizer):
+        self.diarizer = diarizer
+        self.json_outputs = None
+
+    def get_json(self):
+        load_dotenv()
+        self.json_outputs = []
+        i = 0
+        for segment, _, label in self.diarizer.diarization.itertracks(yield_label=True):
```

Comment:
Est-ce que tu as essayé ça ? 
```suggestion
         for count, (segment, _, label) in enumerate(self.diarizer.diarization.itertracks(yield_label=True)):
```

Reply:
Si ça fonctionne, plus besoin du i